dev_tools/scripts/config

```diff
@@ -32,6 +32,12 @@
     - FixInclude: "common/unified/base/kernel_launch_solver.hpp"
 - "(cuda|hip|dpcpp|omp)/base/kernel_launch_solver\."
     - FixInclude: "common/unified/base/kernel_launch_solver.hpp"
+- "dense_kernels.template.cpp"
+    - FixInclude: "core/matrix/dense_kernels.hpp"
+- "/csr_kernels.template.*"
+    - FixInclude: "core/matrix/csr_kernels.hpp"
+- "/fbcsr_kernels.template.*"
+    - FixInclude: "core/matrix/fbcsr_kernels.hpp"
```

Comment:
```suggestion
- "*\.template\.*"
    - PathIgnore: "1"
    - PathPrefix: "core"
```
it should also work

Reply:
ah, does everything past the first `.` get ignored?

Reply:
Actually, it should be `.*\.template\..*`, and it only apply to the csr and fbcsr not common/unified .
the PathIgnore value is the number of folder levels ignored.

You can add the `- CoreSuffix: "\.template"` into the following two sections.

for unified
```
- "common/unified/.*.cpp"
    - PathIgnore: "2"
    - PathPrefix: "core"
```
and for normal kernels
```
- ".*"
    - PathPrefix: "core"
    - PathIgnore: "1"
```
script will ignore the `.template` such that it is like the orignal.

Reply:
thanks for the fix :)